brevbaker/src/main/kotlin/no/nav/brev/brevbaker/InterneDataklasser.kt

```diff
@@ -0,0 +1,13 @@
+package no.nav.brev.brevbaker
+
+import kotlin.RequiresOptIn.Level
+
+@RequiresOptIn(
+    message = "Dette er dataklasser vi bruker internt i brevteamet, mellom applikasjonene våre." +
+            "For deg som konsument anbefaler vi at du heller lager og bruke din egen implementasjon av interfacene vi tilbyr" +
+            "Bruker du likevel våre interne implementasjoner er det på egen risiko, vi endrer disse ved behov uten å annonsere det ut.",
+    level = Level.ERROR
+)
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
```

Comment:
Hva er forskjellen mellom `RUNTIME` og `BINARY` her?

Reply:
KDoc-en her var ganske fin:

/**
 * Contains the list of possible annotation's retentions.
 *
 * Determines how an annotation is stored in binary output.
 */
public enum class AnnotationRetention {
    /** Annotation isn't stored in binary output */
    SOURCE,
    /** Annotation is stored in binary output, but invisible for reflection */
    BINARY,
    /** Annotation is stored in binary output and visible for reflection (default retention) */
    RUNTIME
}